web/src/components/StateDropdown.tsx

```diff
@@ -14,6 +14,7 @@ interface Props {
   validationText?: string;
   excludeNJ?: boolean;
   excludeTerritories?: boolean;
+  includeOutsideUSA?: boolean;
   useFullName?: boolean;
```

Comment:
should we make it consistent for the file and it be excludeOutsideUSA?  I mean either way is fine, its just a question and not a hill for me to die on or even stand on.

Reply:
@cbrunefearless The reason for the include/exclude is to set a reasonable default. The default when passing no props should be: to include NJ and territories, and to NOT include "Outside the USA"

I think it would be weird behavior for the state dropdown to include "Outside the USA" by default and have to explicitly specify for it to be excluded. This behavior is unusual and should be the exception, not the rule